coreos-base/oem-packet/files/phone-home.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+curl -s -o /tmp/jq http://images.packet.net/jq
```

Comment:
I'm open to either, which is preferred? I did it this way because I presumed you wouldn't want some random bash json decoder in the CoreOS codebase.